packages/gatsby-plugin-google-tagmanager/.babelrc

```diff
@@ -0,0 +1,14 @@
+{
+  "presets": [
```

Comment:
gatsby-ssr is not a node environment. It's compiled by webpack and can include webpack stuff like importing files etc.